### PR TITLE
Change autoupdate script

### PR DESCRIPTION
**Pull request by ozlevka-ericom**
https://github.com/EricomSoftwareLtd/SB/issues/2551

### CODE REVIEW

File: Setup/autoupdate.sh [hunk 1]
```diff
@@ -72,34 +72,21 @@ while true; do
...
-                UPDATE=true
-            fi
+
+        if [ "$AM_I_LEADER" == true ]; then
+              if [ "$ES_DEV" == true ]; then
+                $ES_PATH/update.sh update --dev "$FORCE_UPDATE"
+              elif [ "$ES_STAGING" == true ]; then
+                $ES_PATH/update.sh update --staging "$FORCE_UPDATE"
+              else
+                $ES_PATH/update.sh "$FORCE_UPDATE"
```
Comment (BenyH) [inline Setup/autoupdate.sh]: $ES_PATH/update.sh update "$FORCE_UPDATE"

Comment (ozlevka-ericom) [reply Setup/autoupdate.sh]: Why it's more good?
It's not free if...

Comment (BenyH) [reply Setup/autoupdate.sh]: you made a mistake on line 86 is missing the command _update_

**Comment by ozlevka-ericom** [comment]
Change request is done

**Pull request merged**